python/lsst/sims/featureScheduler/basis_functions.py

```diff
@@ -501,6 +505,74 @@ def __call__(self, indx=None):
 
         return result
 
+class Normalized_Target_map_basis_function(Base_basis_function):
```

Comment:
Is this basis function different from `Target_map_basis_function` using the same `norm_factor` parameter?